src/test/java/com/google/sps/servlets/FeedbackServletTest.java

```diff
@@ -0,0 +1,110 @@
+import org.springframework.mock.web.MockHttpServletRequest;
+import org.springframework.test.web.servlet.request.MockHttpServletRequestBuilder;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.junit.Test;
+import com.google.gson.JsonSyntaxException;
+
+@RunWith(JUnit4.class)
+public final class FeedbackServletTest {
+  LocalServiceTestHelper helper =
+      new LocalServiceTestHelper(new LocalCapabilitiesServiceTestConfig());
+  private FakeScheduledInterviewDao scheduledInterviewDao;
+  private final ScheduledInterview scheduledInterview =
+      ScheduledInterview.create(
+          (long) -1,
+          new TimeRange(
+              Instant.parse("2020-07-06T17:00:10Z"), Instant.parse("2020-07-06T18:00:10Z")),
+          "user@company.org",
+          "user@mail.com");
+
+  @Before
+  public void setUp() {
+    helper.setUp();
+    scheduledInterviewDao = new FakeScheduledInterviewDao();
+  }
+
+  @After
+  public void tearDown() {
+    helper.tearDown();
+  }
+
+  // Tests that a valid feedbackServlet request was made.
+  @Test
+  public void validFeedbackServletRequest() throws IOException {
+    FeedbackServlet feedbackServlet = new FeedbackServlet();
+    ScheduledInterviewServlet scheduledInterviewServlet = new ScheduledInterviewServlet();
+    feedbackServlet.init(scheduledInterviewDao);
+    scheduledInterviewDao.create(scheduledInterview);
+    MockHttpServletRequest getRequest = new MockHttpServletRequest();
+    MockHttpServletResponse getResponse = new MockHttpServletResponse();
+    helper.setEnvIsLoggedIn(true).setEnvEmail("user@company.org").setEnvAuthDomain("auth");
+    getRequest.addParameter("interview", "-1");
+    getRequest.addParameter("userTime", "2020-07-05T22:00:00Z");
+    getRequest.addParameter("timeZone", "Etc/UCT");
+    getRequest.addParameter("role", "Interviewer");
+    feedbackServlet.doGet(getRequest, getResponse);
+
+    Assert.assertEquals(200, getResponse.getStatus());
+  }
+
+  // Tests that false is returned when a scheduledInterview id does not exist.
+  @Test
+  public void interviewIdDoesNotExist() throws IOException {
+    FeedbackServlet feedbackServlet = new FeedbackServlet();
+    ScheduledInterviewServlet scheduledInterviewServlet = new ScheduledInterviewServlet();
+    feedbackServlet.init(scheduledInterviewDao);
+    scheduledInterviewDao.create(scheduledInterview);
+    MockHttpServletRequest getRequest = new MockHttpServletRequest();
+    MockHttpServletResponse getResponse = new MockHttpServletResponse();
+    helper.setEnvIsLoggedIn(true).setEnvEmail("user@company.org").setEnvAuthDomain("auth");
+    getRequest.addParameter("interview", "-1");
+    getRequest.addParameter("userTime", "2020-07-05T22:00:00Z");
+    getRequest.addParameter("timeZone", "Etc/UCT");
+    getRequest.addParameter("role", "Interviewer");
+    feedbackServlet.doGet(getRequest, getResponse);
+
+    Assert.assertEquals(false, (boolean) getRequest.getAttribute("feedbackOpen"));
```

Comment:
This does not match the comment for this test. Fix one of them.

Reply:
Done